_posts/en/newsletters/2019-07-10-newsletter.md

```diff
@@ -0,0 +1,116 @@
+    by the spender if the payment succeeds.  ZmnSCPxj describes a method
+    by which they can make their part of the rebalance contingent on
+    them receiving their routing income, ensuring that they either get
+    paid or the rebalance doesn't happen.  This would require additional
+    communication between nodes and so it's a change that probably needs
+    further discussion in order to be considered for addition to the LN
+    specification.
+
+    The second case ZmnSCPxj describes is other people along the
+    rebalance path who themselves want to rebalance one or more of their
+    channels in the same direction as the routing node.  These nodes can
+    allow free routing in that direction to encourage someone to perform
+    that rebalancing.  This second case doesn't require any changes to
+    the LN specification: nodes can already set their routing fees to
+    zero, allowing any other nodes to attempt JIT routing with free
+    rebalances.  The worst case would be that a payment that would've
+    failed anyway will take a bit longer to return a failure message to
+    the spender, a delay equal to the amount of time any routing nodes
+    spent attempting to rebalance their channels in order to support the
+    payment.
+
+## Bech32 sending support
+
+*Week 17 of 24 in a [series][bech32 series] about allowing the people
+you pay to access all of segwit's benefits.*
+
+{% comment %}<!-- weekly reminder for harding: check Bech32 Adoption
+wiki page for changes -->{% endcomment %}
+
+{% include specials/bech32/17-signmessage.md %}
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[LND][lnd repo], [C-Lightning][c-lightning repo], [Eclair][eclair repo],
+[libsecp256k1][libsecp256k1 repo], and [Bitcoin Improvement Proposals
+(BIPs)][bips repo].*
+
+- Bitcoin Core [#15427][Bitcoin Core #15427] and [#16326][bitcoin core
+  #16326] extend the `utxoupdatepsbt` RPC with a `descriptors` parameter
+  that takes an [output script descriptor] and uses it to update a
+  [BIP174][] Partially-Signed Bitcoin Transaction (PSBT) with
+  information about the scripts (addresses) involved in the transaction.
+  This is in addition to the RPC's previous behavior of adding
+  information to the PSBT from the node's mempool and UTXO set.  This
+  new feature is especially useful for hardware wallets and other paired
+  wallets as it makes it possible to add HD key-path information to the
+  PSBTs so that wallets asked to sign a PSBT can easily derive the keys
+  needed for signing or verify that a change output does indeed pay back
+  into the wallet.
+
+- [Bitcoin Core #16257][] aborts adding funds to an unsigned transaction
+  if its feerate is above the maximum amount set by the `maxtxfee`
+  configuration option (default: 0.1 BTC).  Previously, the funding code
+  would silently reduce the fee to the maximum, which merely limited the
```

Comment:
I don't think this captures how dangerous the previous behaviour was. 'merely limiting' the fees to ~$1200 (at time of writing) could be pretty catastrophic. Suggest this is changed to "...reduce the fee to the maximum, which could result in extremely high fees being paid accidentally."